### ISSUE TITLE
No Option To Start Altus Minimized In Tray And Exit Failure Too

### ISSUE THREAD

**Issue by sakobm** [amanharwara/altus#334]
**Describe the bug**
It will be good for add an option where altus starts hidden in the system tray so that it can run during login without showing its window. This option is already available in others like Whatstron and WALC but they don't support multiple accounts. Altus has a start in background option but that option hides it window behind other opened windows when altus is launched. More, there is no way to exit altus (I'm facing this in version 5.3.0) when you set close to tray. When you choose Exit in tray icon, altus fails to exit unless you relogin.

**Information:**
5.3.0 (and maybe others)

**Comment by sakobm** [comment]
> +1, facing the exact same issues, plus like 4-5 instances of Altus I can see in task manager when the fail happens, feels like a mess that leans me into using the official client:(

Lack of tray icon and multiple accounts is what is holding me back. Altus is great, just some few issues

**Comment by SevastianGit** [comment]
I agree, I just really wish they fixed this because it hinders usability a significant bit

**Comment by amanharwara** [comment]
Sorry for this, I don't get a lot of time nowadays to maintain Altus.
I've released a new version [v5.4.1](https://github.com/amanharwara/altus/releases/tag/5.4.1) that should fix this.

**Comment by sakobm** [comment]
> Sorry for this, I don't get a lot of time nowadays to maintain Altus. I've released a new version [v5.4.1](https://github.com/amanharwara/altus/releases/tag/5.4.1) that should fix this.

Okay, I will try that and give feedback.  Thanks soo much
